app/src/main/java/com/techcourse/dao/UserDao.java

```diff
@@ -1,121 +1,55 @@
 package com.techcourse.dao;
 
-import com.techcourse.domain.User;
-import nextstep.jdbc.JdbcTemplate;
+import java.util.List;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.List;
+import com.techcourse.domain.User;
+
+import nextstep.jdbc.JdbcTemplate;
+import nextstep.jdbc.RowMapper;
 
 public class UserDao {
 
     private static final Logger log = LoggerFactory.getLogger(UserDao.class);
 
-    private final DataSource dataSource;
+    private static final RowMapper USER_ROW_MAPPER = (rs, rowNum) ->
```

Comment:
```suggestion
    private static final RowMapper<User> USER_ROW_MAPPER = (rs, rowNum) ->
```
`RowMapper`가 제네릭을 사용한 타입 제공을 해주는 상태라면 이렇게 선언해주는게 나을 것 같네요 :)

그렇게되면 하단에 있는 메소드들에 return 값에 타입변환 (ex `(User)`, `(List<User>)`)도 제거되는 효과를 가져올 수 있겠네요

Reply:
와 너무 좋은 생각입니다!! 수정했습니다 :)